tenable/io/v3/explore/findings/api.py

```diff
@@ -108,3 +104,237 @@ def search_webapp(self,
+                    >>> {
+                    ...  'or': [{
+                    ...      'and': [{
+                    ...              'value': '1',
+                    ...              'operator': 'oper',
+                    ...              'property': '1'
+                    ...          },
+                    ...          {
+                    ...              'value': '2',
+                    ...              'operator': 'oper',
+                    ...              'property': '2'
+                    ...          }
+                    ...      ]
+                    ...  }],
+                    ...  'and': [{
+                    ...      'value': '3',
+                    ...      'operator': 'oper',
+                    ...      'property': 3
+                    ...  }]
+                    ... }
+            sort (list[tuple], optional):
+                A list of dictionaries describing how to sort the data
+                that is to be returned.
+                Examples:
+                    >>> [('field_name_1', 'asc'),
+                    ...      ('field_name_2', 'desc')]
+            limit (int, optional):
+                Number of objects to be returned in each request.
+                Default and max_limit is 200.
+            next (str, optional):
+                The pagination token to use when requesting the next page of
+                results. This token is presented in the previous response.
+            return_resp (bool, optional):
+                If set to true, will override the default behavior to return
+                a requests.Response Object as is to the user.
+            return_csv (bool, optional):
+                If set to true, it will return the CSV response or
+                iterable (based on return_resp flag). Iterator returns all
+                rows in text/csv format for each call with row headers.
+        Returns:
+            Iterable:
+                The iterable that handles the pagination for the job.
+            requests.Response:
+                If ``return_resp`` is set to ``True``, then a response
+                object is returned instead of an iterable.
+        Examples:
+            >>> tio.v3.explore.findings.search_host_audit(
+            ... fields=['finding_id'], limit=2)
+        '''
+        iclass = SearchIterator
+        if kw.get('return_csv', False):
+            iclass = CSVChunkIterator
+        return super()._search(iterator_cls=iclass,
+                               sort_type=self._sort_type.property_based,
+                               resource='findings',
+                               api_path=f'{self._path}/host_audit/search',
+                               **kw
+                               )
+
+    def search_host(self,
+                    **kw
+                    ) -> Union[SearchIterator, CSVChunkIterator, Response]:
+        '''
+        Search and retrieve the Host Resource Vulnerabilities based on supported conditions.
```

Comment:
Update below line -
`Search and retrieve the Host Resource Vulnerabilities based on supported conditions.`
to this -
`Search and retrieve the Host Vulnerabilities based on supported conditions.`